pkg/dynamicinstrumentation/uploader/stack_trace.go

```diff
@@ -114,6 +114,9 @@ func pcToLine(procInfo *ditypes.ProcessInfo, pc uint64) (*funcInfo, error) {
 		return nil, fmt.Errorf("invalid file number in dwarf function entry associated with compile unit")
 	}
 
+	if int(fileNumber) >= len(files) || files[fileNumber] == nil {
+		return nil, fmt.Errorf("could not find file")
```

Comment:
consider writing the index (`fileNumber` and `len(files)` in the log message

Reply:
The file number and files slice are DWARF specific, they're not particularly useful for the sake of debug-ability.